components/map/components/layer-manager/component.jsx

```diff
@@ -32,7 +32,7 @@ class LayerManagerComponent extends PureComponent {
           }
         : null;
 
-    const allLayers = layers.filter((l) => l);
+    const allLayers = [basemapLayer, ...layers].filter((l) => l);
```

Comment:
so, if I understand correctly sending the basemapLayer alongside the array of layers doesn't affect the final result, right?

Reply:
You're right, it doesn't affect the result. What we're defining as basemaps are really just regular layers (in this case satellite imagery ones). 

The only thing we need to pay attention to is the [zIndex](https://github.com/wri/gfw/blob/develop/components/map/components/layer-manager/component.jsx#L31); it needs to be lower than any other layers'; the layer manager will order them by zIndex. zIndex for the layers is assigned [here](https://github.com/wri/gfw/blob/develop/components/map/selectors.js#L432-L452).

We'd usually add the basemap layer first (in the array is fine), because layers are usually stacked on top of each other depending of the order they are added in, but because we're using `zIndex`'s we don't have to worry too much about it.